src/main/java/org/openhab/tools/analysis/tools/CheckstyleChecker.java

```diff
@@ -113,7 +113,7 @@ public void execute() throws MojoExecutionException {
         }
 
         // Maven may load an older version, if no version is specified
-        Dependency checkstyle = dependency("com.puppycrawl.tools", "checkstyle", "7.7");
+        Dependency checkstyle = dependency("com.puppycrawl.tools", "checkstyle", "8.1");
```

Comment:
Yes you are right, I couldn't find other solution. In this case maybe it is better to leave it as it is now.